maintenance/monitor/monitor-component-visual.md

```diff
@@ -171,6 +171,8 @@ After login you will be prompted to change the default password, you can either
 
 Congratulation you have arrived at your Felix dashboard.
 
+![]({{ site.baseurl }}/images/grafana-dashboard.png)
```

Comment:
Maybe we should move the screenshot to be earlier in the doc. With the screenshot where it is currently, by the time the user gets to this final step they already have the dashboard, so the screenshot doesn't add much value for them.  Whereas if we had it earlier then people would more quickly understand what the guide is about. e.g. Perhaps at the beginning of the how-to section, to show people what the guide will get them.  "This how-to will guide you through the steps to generate the following dashboard:" or similar?

Reply:
good point, I have relocated the image to `Value` section with a sentence to accompany it.